blogposts/2024-05-15-improved-gallery-fabric-0.74.md

```diff
@@ -0,0 +1,107 @@
+
+### CoreApp APIs and prototypes
+
+CoreApp was created as a testing ground to explore potential directions for Fabric. We've taken these learnings back into the main Fabric app templates, and have therefore deprecated the CoreApp option for Paper apps. 
+
+## Reliability Boost
+
+While perhaps less flashy, another crucial aspect of 0.74 is our commitment to enhancing the reliability and trustworthiness of React Native Windows. In this release, we've tackled numerous bugs, ranging from resolving [ScrollView's momentum scroll](https://github.com/microsoft/react-native-windows/commit/1367b74599cc888082d280d728b1ac02917cea96) issues and [preventing memory leaks from HTTP responses](https://github.com/microsoft/react-native-windows/commit/ea58fe084c5d8a9116c69807ca9473aaa9e54221) to addressing [UIScheduler deadlocking during shutdown](https://github.com/microsoft/react-native-windows/commit/e2bf3e27f1ad328bcc74d00083482ed5d74a4af6).
+
+If you encounter any bugs in 0.74, please help us out by filing an issue on our [repository](https://github.com/microsoft/react-native-windows/issues)!
+
+# ✨ Improved [React Native Gallery](https://www.microsoft.com/store/productId/9NPG0B292H4R?ocid=pdpshare) App
+
+We've been hard at work revitalizing our Gallery app, aligning it with the [WinUI Gallery](https://aka.ms/winuiGallery) to showcase a more sleek and modern RNW application. In addition, we've ensured it meets inclusive and accessible design requirements and broadened the array of examples to encompass more community modules and common scenarios.
+
+Get React Native Gallery from the [Microsoft store](https://www.microsoft.com/store/productId/9NPG0B292H4R?ocid=pdpshare) or get the latest changes from our (Github repository)[https://github.com/microsoft/react-native-gallery]!
+
+![rn-gallery-gif-2024](assets/gallery-demo-2024.gif)
+
+# 🗺️ Roadmap to Fabric
+
+We've committed ourselves to adding Windows support for React Native's [Fabric](https://reactnative.dev/architecture/fabric-renderer) architecture, the latest rendering system for React Native. Designed for efficiency across platforms in C++, React Native for Windows Fabric seamlessly targets Composition while allowing hosting XAML islands within. Fabric-powered apps default to [WinAppSDK](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/) architecture, aligning with modern [WinUI3 standards](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/). Stay tuned for migration guidance for current UWP RNW apps. Follow our journey to Fabric [here](https://github.com/microsoft/react-native-windows/issues/12042).
+
+While our migration efforts to Fabric are underway, our applications aren't quite ready for publication just yet. But fear not! We're thrilled to offer you a sneak peek at a functional app in both Paper and Fabric versions. Please bear in mind that Fabric may exhibit some visual disparities that we are working to iron out.
```

Comment:
> our applications aren't quite ready for publication just yet

Something about this feels not exactly phrased right. The platform isn't ready for production apps. When we say "our applications" here... which applications does that refer to? Bit of stumbling point for me, but minor.

Reply:
fixed!